### PR TITLE
Replaced all instances of Account Manager with Wallet

### LINKED ISSUE #1843: Replace all instances of "Account Manager" with "Wallet"
Since we are rebranding the "Account Manager" to "Wallet" we can replace all instances on the website. For example:

Screen Shot 2021-06-09 at 10 12 56 PM

### PR DESCRIPTION
**Pull request by avinashupadhya99**
Closes #1843 

Account number - `94cf53921e9c5ddf48dc5589b7cdf111f7acdc93dc3073b94880416d4676b9ae`

### CODE REVIEW

File: src/types/faq.tsx [hunk 1]
```diff
@@ -110,7 +110,7 @@ export const faqQuestionsAndAnswers: TopicQuestionAndAnswers[] = [
         ),
         id: 'where-download-account-manager',
```
Comment (jamessspanggg) [inline src/types/faq.tsx:111]: can change the ids as well?

File: src/types/faq.tsx [hunk 2]
```diff
@@ -212,7 +212,7 @@ export const faqQuestionsAndAnswers: TopicQuestionAndAnswers[] = [
           </span>
         ),
         id: 'account-manager-guides',
```
Comment (jamessspanggg) [inline src/types/faq.tsx:214]: here too

**Comment by avinashupadhya99** [comment]
@jamessspanggg do you want me to update the routes as well?

**Comment by angle943** [comment]
@avinashupadhya99 yes update the routes as well please

**Comment by angle943** [comment]
pretty much there should be absolute no mention of the Account manager in the codebase whatsoever

**Review by angle943 — CHANGES_REQUESTED** [review]
pretty much there should be absolute no mention of the Account manager in the codebase whatsoever

**Comment by avinashupadhya99** [comment]
I have not updated the references to the Account Manger repo since it is still named Account Manager

**Review by jamessspanggg — APPROVED** [review]

**Comment by jamessspanggg** [comment]
Thanks for the help! Raised the bounty for ya @avinashupadhya99

**Pull request merged**